### ISSUE TITLE
How to assign prediction to timeseries in NBeats? 

### ISSUE THREAD

**Issue by DonBeo** [jdb78/pytorch-forecasting#527]
Hi,
thanks for the great library. 
I have an usability question. 

I have trained an NBeats model on a dataframe containing 16 time series as in the example (https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/ar.html)
 
Now I am predicting new data from a dataframe as:
```
pr, x_pr, idx = best_model.predict(data_ts, mode="prediction", return_x=True, return_index=True)
```
as expected I get prediction for 16 time series and 192 points (this was my prediction length)

`pr.shape #= torch.Size([16, 192])`

Now how do I assign each row in the prediction to the correct time series? 
In other words I would like to know to which of my 16 time series each row refers to.  In my data the time series is identified by the `series` column as in  the attached figure

![image](https://user-images.githubusercontent.com/4463692/119977454-bebee000-bfb8-11eb-9658-12f57606836a.png)

**Comment by jdb78** [comment]
You can use `idx` here to match them to the timeseries.

**Comment by DonBeo** [comment]
thanks for answering.  So each row of the `idx` dataframe returned by the function corresponds to a row of `pr`

I really suggest to add it to the doc as it was not very clear. 

Thanks for the work you are doing!!

**Comment by jdb78** [comment]
Adding in the latest PR.
